src/App.tsx

```diff
@@ -127,19 +128,8 @@ function App() {
             }}
           />
         </Route>
-        <Route path="/:language/Canada">
-          <TableauEmbed
-            url={CANADA_URLS}
-            key={`CanadianTableau${language}`}
-            desktopOptions={{
-              width: "80vw",
-              height: "4100px"
-            }}
-            mobileOptions={{
-              width: "90vw",
-              height: "2800px"
-            }}
-          />
+        <Route path="/:language/Studies/:name">
```

Comment:
I think "/Partnerships" would be more suitable for this. What do you think @jordanvanwyk @rahularoradfs